seqr/views/apis/saved_variant_api_tests.py

```diff
@@ -794,6 +794,19 @@ def test_update_variant_main_transcript(self):
         saved_variant = SavedVariant.objects.get(guid=VARIANT_GUID)
         self.assertEqual(saved_variant.selected_main_transcript_id, transcript_id)
 
+    def test_update_variant_acmg_classification(self):
+        update_variant_acmg_classification_url = reverse(update_variant_acmg_classification_handler, args=[VARIANT_GUID])
+        acmg_classification = {
+            'classify': 'Uncertain',
+            'criteria': ['PM2_P'],
+            'score': 1
+        }
+        # self.check_manager_login(update_variant_acmg_classification_url, request_data={'acmgClassification': acmg_classification})
```

Comment:
this endpoint is authenticated to anyone with read access to the project, so you should be testing this with `self.check_collaborator_login` and not `self.check_manager_login`. Also, the endpoint should be able to check access without the request_body, so you should just be running `self.check_collaborator_login(update_variant_acmg_classification_url)`